runners/flink/flink_runner.gradle

```diff
@@ -148,6 +148,8 @@ dependencies {
   compile "org.apache.flink:flink-java:$flink_version"
   compile "org.apache.flink:flink-runtime_2.11:$flink_version"
   compile "org.apache.flink:flink-streaming-java_2.11:$flink_version"
+  // RocksDB state backend
+  compile "org.apache.flink:flink-statebackend-rocksdb_2.11:$flink_version"
```

Comment:
Just wanted to confirm that dependency won't be baked into the job server as it is already part of the Flink dist.

Reply:
Is that true? AFAIK RocksDB is an optional dependency. We can change the scope to `provided` and add a separate dependency with `testCompile`.

Reply:
The dependency is part of the flink distribution:
```
$ tar -tf flink-dist_2.12-1.11.2.jar | grep RocksDBStateBackend
org/apache/flink/contrib/streaming/state/RocksDBStateBackend.class
org/apache/flink/contrib/streaming/state/RocksDBStateBackendFactory.class
org/apache/flink/contrib/streaming/state/RocksDBStateBackend$PriorityQueueStateType.class
```
If it wasn't, you could not configure the default state backend:
https://ci.apache.org/projects/flink/flink-docs-stable/ops/state/state_backends.html#setting-default-state-backend

Reply:
RocksDB used to be not included in the dist. Good point, we can set it to provided then.

Reply:
I've verified that RocksDB is not included in any of the to-be-released jars.